lab-aaron/src/component/category-form/index.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+
+class CategoryForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: props.category ? props.category.title : '',
+      id: props.category ? props.category.id : null,
+      timestamp: props.category ? props.category.timestamp : null,
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentDidMount() {
+    console.log('__FORM_PROPS__', this.props);
+  }
+
+  handleChange(e) {
+    this.setState({ title: e.target.value });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    this.props.onComplete(Object.assign({}, this.state));
+    this.setState({title: ''});
+  }
+
+  render() {
+    return (
+      <form className="category-form" onSubmit={this.handleSubmit}>
+        <input
+          required
+          type="text"
+          name="name"
```

Comment:
whatever is inside of name="name" (in this case, you've called it name) needs to match the value exactly, which needs to be a prop on the state of that component, so 
```
name="title"
value={this.state.title}
```

Reply:
Awesome Thanks!